install.sh

```diff
@@ -222,6 +222,7 @@ function install_mcu_templates {
             filename=$(basename "${file_list[$((ercf_template-1))]}")
             cat "${FRIX_CONFIG_PATH}/user_templates/mcu_defaults/ercf/$filename" >> ${USER_CONFIG_PATH}/mcu.cfg
             printf "[CONFIG] Template '$filename' inserted into your mcu.cfg user file\n\n"
```

Comment:
As you added a new line just after this one, you will also need to change it regarding the `\n` at the end of the line.
So can you change this line to:
```
echo "[CONFIG] Template '$filename' inserted into your mcu.cfg user file"
```